gunicorn_custom_workers.py

```diff
@@ -1,8 +1,18 @@
 import gevent
 from gunicorn.workers.ggevent import GeventWorker
+from psycogreen.gevent import patch_psycopg
 
 
 class MyGeventWorker(GeventWorker):
+    def patch_psycopg2(self):
+        patch_psycopg()
+        self.log.info("Patched Psycopg2 for gevent")
+
+    def patch(self):
+        super().patch()
+        self.log.info("Patched all for gevent")
+        self.patch_psycopg2()
+
```

Comment:
That's right, to prevent any issue I would like the 2 patches to be as close as posible. We are not currently having any issue but I prefer to make sure we won't in the future either. So as we are already extending the worker it makes sense to me to make the patching here